packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -57,29 +57,29 @@ describe("<DatePicker />", () => {
     expect(screen.getByDisplayValue("Mar 1, 2022")).toBeInTheDocument()
   })
 
-  it("allows you to tab through input, button and calendar", () => {
+  it("allows you to tab through input, button and calendar", async () => {
```

Comment:
Oh I experimented with using

```js
expect(await screen.findByRole("dialog")).toBeVisible()
```

instead of:

```js
waitFor(() => {
  expect(screen.getByRole("dialog")).toBeVisible()
})
```

What do you think? Better to always use `waitFor` rather than the more subtle `findByRole`?

Reply:
Hmm... I think it reads slightly clearer when we use `waitFor`. That being said, I don't really have an issue with either. I think it depends on what might be deemed the better practice? Perhaps one is more performant than the other? Might be worth a quick google to see. I definitely know when I was looking up things recently that I tended to see suggestions using `waitFor` and not many that suggest `await findBy`.